src/util/options.schema.json

```diff
@@ -1669,6 +1697,12 @@
           "type": "boolean",
           "default": true
         },
+        "float": {
+          "title": "warn.float",
+          "description": "float warnings",
+          "type": "boolean",
+          "default": false
```

Comment:
As we didn't want to break any current behaviour, we disabled all our introduced options by default so that the current behaviour of goblint does not change. However, we could enable the warnings as they are also additionally _guarded_ by the float analysis itself.